### PR TITLE
Update Helix.SDK to consume fix on Download files from results container task

### PR DESCRIPTION
**Pull request by kunalspathak**
Motivation to update it is to bring the fix in https://github.com/dotnet/arcade/pull/6179/.

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -10,61 +10,61 @@
...
+      <Sha>a8d7ecca110a9932c8ee7fc9e1065b6ad7c96dfd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.VersionTools.Tasks" Version="5.0.0-beta.20459.8">
+    <Dependency Name="Microsoft.DotNet.VersionTools.Tasks" Version="5.0.0-beta.20472.5">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>91470b0b14ba016c1fb78211b12775287c17b34e</Sha>
+      <Sha>a8d7ecca110a9932c8ee7fc9e1065b6ad7c96dfd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" Version="5.0.0-beta.20459.8">
+    <Dependency Name="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" Version="5.0.0-beta.20472.5">
```
Comment (safern) [inline eng/Version.Details.xml:65]: Let's revert all changes to this file except for the `Microsoft.DotNet.Helix.Sdk` entry

Comment (kunalspathak) [reply eng/Version.Details.xml:65]: sure.

**Review by safern — APPROVED** [review]

**Comment by kunalspathak** [comment]
@safern - I have rebased the changes. Please take another look.

**Comment by safern** [comment]
@kunalspathak I think we need to fix conflicts again 😢

**Comment by kunalspathak** [comment]
> @kunalspathak I think we need to fix conflicts again 😢

I don't think this PR is needed anymore. The commit already includes the required fixes.

**Pull request closed without merging**